capture.py

```diff
@@ -741,7 +760,12 @@ def _isolated_nodes(nodes, panel):
         cmds.isolateSelect(panel, state=True)
         for obj in nodes:
             cmds.isolateSelect(panel, addDagObject=obj)
-    yield
+
+    try:
+        yield
+    finally:
+        if nodes is not None:
+            cmds.isolateSelect(panel, state=False)
```

Comment:
This doesn't actually preserve the original state - it just disables isolate view after the context. Shouldn't this revert it back to the state it was at the beginning of the context - as such.. shouldn't it contain the same True/False state *and* the same nodes?